histogrammar/resources.py

```diff
@@ -19,22 +19,19 @@
 
 
 # Resources lookup file for histogrammar
-
-import pathlib
-from pkg_resources import resource_filename
-import histogrammar as hg
+from importlib import resources
 
 
 # data files that are shipped with histogrammar.
 _DATA = {
     _.name: _
-    for _ in pathlib.Path(resource_filename(hg.__name__, "test_data")).glob("*")
+    for _ in resources.files("histogrammar.test_data").iterdir()
```

Comment:
I think I'd go for using module object inside `resource.files(xxxx)`, not a string with module name. Then the IDE would more easily catch the reference in case someone e.g. renames the test_data module:

```
from histogrammar import test_data, notebooks

# data files that are shipped with histogrammar.
_DATA = {
    _.name: _
    for _ in resources.files(test_data).iterdir()
}

# Tutorial notebooks
_NOTEBOOK = {
    p.name: p
    for p in resources.files(notebooks).iterdir() if p.suffix == ".ipynb"
}

```

Reply:
Fixed.